### PR TITLE
feat(route): add App Center Release

### PR DESCRIPTION
**Pull request by Rongronggg9**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/app-center/release/gameonline/baitomobile/baito
/app-center/release/cloudflare/1.1.1.1-windows/beta
/app-center/release/rdmacios-k2vy/microsoft-remote-desktop-for-mac/all-users-of-microsoft-remote-desktop-for-mac
/app-center/release/rafalense-70ux/plus-release/public
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [x] New Route
- [x] Documentation
  - [x] CN
  - [x] EN
- [x] 全文获取 fulltext
  - [x] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] 日期和时间 date and time
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

### CODE REVIEW

File: docs/en/program-update.md [hunk 1]
```diff
@@ -16,6 +16,20 @@ pageClass: routes
 
 <Route author="nczitzk" example="/anytxt/release-notes" path="/anytxt/release-notes"/>
 
+## App Center
+
+### Release
+
+<Route author="Rongronggg9" example="/app-center/release/cloudflare/1.1.1.1-windows/beta" path="/app-center/release/:user/:app/:distribution_group" :paramsDesc="['User', 'App name', 'Distribution group']" radar="1" rssbud="1">
```
Comment (TonyRL) [inline docs/en/program-update.md:23]:
```suggestion
<RouteEn author="Rongronggg9" example="/app-center/release/cloudflare/1.1.1.1-windows/beta" path="/app-center/release/:user/:app/:distribution_group" :paramsDesc="['User', 'App name', 'Distribution group']" radar="1" rssbud="1">
```

File: docs/en/program-update.md [hunk 2]
```diff
@@ -16,6 +16,20 @@ pageClass: routes
...
+
+::: tip
+
+The parameters can be extracted from the Release page URL: `https://install.appcenter.ms/users/:user/apps/:app/distribution_groups/:distribution_group`
+
+:::
+
+</Route>
```
Comment (TonyRL) [inline docs/en/program-update.md:31]:
```suggestion
</RouteEn>
```

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: docs/en/program-update.md (hunk 1 above)
Comment (Rongronggg9) [reply docs/en/program-update.md]: done

File: docs/en/program-update.md (hunk 2 above)
Comment (Rongronggg9) [reply docs/en/program-update.md]: done

**Pull request merged**